provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -148,15 +147,7 @@ def _get_pool_property(self, pool: PoolHandle, prop_name: str) -> int:
                      f'durability>{pool_type}>{prop_name}'))
 
     def _get_layout(self, pool: PoolHandle) -> Optional[Layout]:
-        conf = self.provider
         (cluster_id, pool_type, storage_ndx) = pool.tuple()
-        # cluster>storage_set[N]>durability>{type}
-        type_value = conf.get(
-            f'cluster>storage_set[{storage_ndx}]'
-            f'>durability>{pool_type}',
-            allow_null=True)
-        if not type_value:
```

Comment:
Are we not using `type_value ` anymore?
I see if the `type_value` for fetching pool type is missing then we directly return None.